source/plugin/text/representations/index.rst

```diff
@@ -12,6 +12,7 @@ applicable formats:
     formatting-code-legacy
     xml
     json
+    configurate
 
 The ``org.spongepowered.api.text.serializer.TextSerializers`` class provides three ``TextSerializer``\ s for serializing
```

Comment:
The `TextConfigSerializer` does not extend `TextSerializer` but rather `TypeSerializer` so indicating that it does here would be incorrect.